cuML/CMakeLists.txt

```diff
@@ -79,27 +89,60 @@ list(GET GPU_ARCHS -1 ptx)
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode arch=compute_${ptx},code=compute_${ptx}")
 ## end nvcc options
 
+###################################################################################################
+# - include paths ---------------------------------------------------------------------------------
+
 include_directories(src
-  ${GTEST_DIR}/googletest/include
-  ${MLPRIMS_DIR}/src
-  ${MLPRIMS_DIR}/src/dbscan
-  ${MLPRIMS_DIR}/test
-  ${CUTLASS_DIR}
-  ${CUTLASS_DIR_2}
-  ${CUB_DIR})
+                    ${GTEST_DIR}/googletest/include
```

Comment:
In fact within CMake it's possible append even more. `include_directories(SYSTEM ${GTEST_DIR}/googletest/include)` command will append this directory for search path for all targets that will be created after this command. But...It will turn off all warning which happend. So in my opinion it can be usefull to just turn off warning from all 3rd party libraries from rapids.